src/calculateRentalCost.js

```diff
@@ -1,10 +1,13 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
```

Comment:
Prefer using `const` for the price variable since it's not being reassigned, but modified. This helps to prevent unintentional reassignments.